source/assets/stylesheets/style.css.sass

```diff
@@ -2,7 +2,7 @@
 // ========================================
 
 // Rolodex gem
-@import rolodex
+@import vendor/rolodex
```

Comment:
Is this because v4 of middleman removes the asset pipeline? This is probably a dumb question, but how is the `Sass` getting compiled?